components/expenses/ProcessExpenseButtons.js

```diff
@@ -132,6 +132,7 @@ const ProcessExpenseButtons = ({
   onDelete,
   isMoreActions,
   displaySecurityChecks,
+  displayApproveExpense,
 }) => {
   const [selectedAction, setSelectedAction] = React.useState(null);
   const onUpdate = (cache, response) => onSuccess?.(response.data.processExpense, cache, selectedAction);
```

Comment:
Should we use the `useProcessExpense` hook here as well?

Reply:
I was thinking that, I tried to experiment with this pattern, it seemed appropriate to abstract some of that logic in a hook. I will try to refactor this piece as well.